doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -0,0 +1,74 @@
+[id='exporting-importing-dashbuilder-data-proc-{context}']
+= Exporting and importing Dashbuilder data
+
+Dashbuilder is a dashboard and reporting tool integrated in {CENTRAL} and is used by the Datasets editor and Content Manager page. There are three data types:
+
+* Datasets
+* Perspectives
+* Navigation
+
+You can import and export Dashbuilder data as ZIP files in {CENTRAL}.
+
+NOTE: This feature can only be accessed by administrator users.
```

Comment:
Hey @csherrar,
AFAIK if this note is must have then instead of NOTE can we make it as IMPORTANT? 
Because NOTEs can be missed out easily. 
Also, I have slightly tweaked the sentence here. 
**Suggestion:** This feature is accessible only to the administrator users.

Reply:
@kaldesai I like the suggestion. I will make that change.

Reply:
@kaldesai I reworded it slightly to "IMPORTANT: This feature is only accessible by administrator users."